src/main/java/com/team766/controllers/PIDSTATE.java

```diff
@@ -0,0 +1,7 @@
+package com.team766.controllers;
+
+	public enum PIDSTATE {
```

Comment:
you don't really need to all-caps this, `PIDState` is fine

Reply:
When I use enums, my practice is to capitalize the name. Let me know if that is bad.

Reply:
https://howtodoinjava.com/java/enum/enum-tutorial/

Heres an online reference on how to style enums

Reply:
okay, fixing now